lib/audit/moab_to_catalog.rb

```diff
@@ -65,4 +65,11 @@ def self.check_existence_for_all_storage_roots
       Rails.logger.info end_msg
     end
   end
+
+  def self.drop_storage_root(endpoint_name)
```

Comment:
this should probably be named drop_endpoint ??  (as we could conceivably use it for archival endpoints too?)